RedHat-UX/red-hat-brand
# fix(card): add flex variant which pins the CTA to the bottom

## What I did

1. Added a flex variant styles to rhb-card.webc component

Closes #13 

## Testing instructions

1. View [deploy preview typography page ](https://deploy-preview-17--prismatic-souffle-43beb6.netlify.app/design-language/typography/) and see the card content for "functional type and expressive type"
